boards/arm/qemu_cortex_m0/Kconfig.defconfig

```diff
@@ -32,9 +32,6 @@ config SYS_CLOCK_HW_CYCLES_PER_SEC
 config SYS_CLOCK_TICKS_PER_SEC
 	default 100
 
-config ENTROPY_NRF_FORCE_ALT
-	default y
```

Comment:
This is not a redefinition of the `ENTROPY_NRF_FORCE_ALT` Kconfig symbol, but a default configuration specification.

Note that `ENTROPY_NRF_FORCE_ALT` is not `default y` in `Kconfig.nrf5`.

Reply:
> > QEMU Cortex-M0 Board includes a system timer driver implementation based on the nRF TIMER peripheral, because the RTC peripheral is not supported in QEMU.
> 
> @ioannisg So I suppose this should not be necessary if we implement nRF RTC peripheral support in the QEMU?

Correct.

Reply:
Yeap, I cleaned the commit message now, thanks @stephanosio